build.json

```diff
@@ -216,6 +216,7 @@
       ],
       "src": [
         "src/core/census/grpc_context.c",
+        "src/core/channel/census_filter.c",
```

Comment:
Good point, I see security filters are in core/security directory, maybe census_filters should also be in census directory. I'll let you decide, Craig.